swan/python/configure_kernels_and_terminal.py

```diff
@@ -56,8 +56,6 @@ def configure_terminal():
                 # Setting up colors
                 "alias ls='ls --color'\n",
                 "alias grep='grep --color'\n",
-                # FIXME: Fix this workaround and load the correct user home
-                # correctly
                 f"cd {HOME}\n"
```

Comment:
Instead of removing the comment, I would replace it with what is in the commit.

Also, expectable behaviour should be in quotes, since people who use Lab but already expect otherwise.

Reply:
Addressed, please final approve.